src/sonic-config-engine/sonic-cfggen

```diff
@@ -47,7 +48,9 @@ def unique_name(l):
 
 def main():
     parser=argparse.ArgumentParser(description="Render configuration file from minigraph data and jinja2 template.")
-    parser.add_argument("-m", "--minigraph", help="minigraph xml file")
+    group = parser.add_mutually_exclusive_group()
```

Comment:
No, it's not true that one of them is required.

---
In reply to: [125150833](https://github.com/Azure/sonic-buildimage/pull/775#discussion_r125150833) [](ancestors = 125150833)